### PR TITLE
Introduce replicas options

### PR DESCRIPTION
**Pull request by kvaps**
- Add options to specify replicas for every application
- Fix `versions_map` generator script
- Specify correct versions of applications for every packege

### CODE REVIEW

File: packages/apps/http-cache/values.yaml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 external: false
+
+haproxy:
+  replicas: 2
+nginx:
+  replicas: 3
```
Comment (themoriarti) [inline packages/apps/http-cache/values.yaml]: Why do you need three replicas? two replicas are enough for minimum fault tolerance.

File: packages/apps/redis/values.yaml [hunk 2]
```diff
@@ -1,2 +1,3 @@
+replicas: 3
```
Comment (themoriarti) [inline packages/apps/redis/values.yaml]: Why three replicas? 2 is enough for redis master-slave

File: packages/apps/http-cache/values.yaml (hunk 1 above)
Comment (kvaps) [reply packages/apps/http-cache/values.yaml]: fixed, thank you

File: packages/apps/redis/values.yaml (hunk 2 above)
Comment (kvaps) [reply packages/apps/redis/values.yaml]: fixed, thank you

**Review by themoriarti — APPROVED** [review]
Practically everything is fine, I did not look into the VPN replicas, but we will fix them in the next PR #110

**Pull request merged**